### PR TITLE
Program Status Changes

### PR DESCRIPTION
**Pull request by cesar82**

### CODE REVIEW

**Review by fabian202 — APPROVED** [review]

File: src/components/atoms/Comment/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
```
Comment (normandanielmq) [inline src/components/atoms/Comment/index.js:1]: `;` are no longer needed

Comment (normandanielmq) [reply src/components/atoms/Comment/index.js:1]: This component does not look like as an `atom`

File: src/components/atoms/Comment/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  position: absolute;
+  top: 2rem;
+  right: 0;
+  display: none;
+  button {
+    margin-right: 0;
+    padding: 0;
+  }
+`;
+
```
Comment (normandanielmq) [inline src/components/atoms/Comment/index.js:65]: Remember to put styles on another `style.js` file

File: src/components/molecules/CommentForm/index.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { Form, Button as Btn } from 'react-bootstrap';
+
+const CommentForm = ({ onHide, text }) => {
+  const [comment, setComment] = useState();
```
Comment (normandanielmq) [inline src/components/molecules/CommentForm/index.js:5]: Why `useForm` were not included here?

File: src/components/pages/ProgramDetailsPage/index.js [hunk 4]
```diff
@@ -74,6 +78,18 @@ const ProgramDetails = ({ id }) => {
...
+      const users = dataUsers.map(
+        ({ id, fullName }) => ({
+          value: id,
+          label: fullName,
+        })
+      );
+      setUserList(users);
+    }
+  }, [dataUsers]);
+
```
Comment (normandanielmq) [inline src/components/pages/ProgramDetailsPage/index.js:92]: Do we really need this transformation? dropdowns have the option to specify what key must be used for the label and which one for the value

File: src/hooks/useProgram/deserializeForm.js [hunk 5]
```diff
@@ -36,8 +36,9 @@ export default function (
     descriptiveElementsStepComplete,
     eligibilityCodingStepComplete,
     recheck,
+    assignedUser,
   } = form;
-
+  console.log(`assignedUser`, assignedUser)
```
Comment (normandanielmq) [inline src/hooks/useProgram/deserializeForm.js:41]: Remove

**Pull request merged**